client/my-sites/site-settings/composing/style.scss

```diff
@@ -0,0 +1,7 @@
+.composing__module-settings.is-indented {
+	margin: 16px 32px 0;
```

Comment:
@MichaelArestad this has been implemented in 76aa0071685542ed57d5f37cee65fcde051647f2.